parlai/crowdsourcing/tasks/turn_annotations_static/turn_annotations_blueprint.py

```diff
@@ -215,15 +226,28 @@ def process_data(self, data_dicts, annotation_indices=None):
                     )
                 for a in annotation_indices[conv_idx]:
                     processed_dialog = self._process_conversation(d, [a])
+                    total_annotation_count += 1
                     output.append(processed_dialog)
+            elif self.annotation_last_only:
+                # Annotate only the last utterance
+                last_idx = len(d['dialog']) * 2 - 1
+                processed_dialog = self._process_conversation(
+                    d, annotation_indices=[last_idx]
+                )
+                total_annotation_count += 1
+                output.append(processed_dialog)
             else:
+                # Annotate all the model utterances
+                total_annotation_count += len(d['dialog']) / 2
                 processed_dialog = self._process_conversation(
                     d, annotation_indices=None
                 )
                 output.append(processed_dialog)
+
         print(
-            f'Processed {len(data_dicts)} total conversations into {len(output)} conversations to be used in crowdsourcing task with {total_annotation_count} total annotations.'
+            f'process_data: Processed {len(data_dicts)} total conversations into {len(output)} conversations in the full data with {total_annotation_count} total turn annotations. (Does not account for units per assignment values.)'
```

Comment:
Hmm can you clarify "(Does not account for units per assignment values.)" further?